ci-operator/step-registry/workers-rhel/ibmcloud-provision/workers-rhel-ibmcloud-provision-commands.sh

```diff
@@ -52,7 +53,9 @@ ${IBMCLOUD_CLI} is keys --all-resource-groups
 rhelSSHKey="$(cat ${CLUSTER_PROFILE_DIR}/ibmcloud-sshkey-name)"
 
 #use the same sgs created by installer, shared with the default rhcos nodes
+run_command "${IBMCLOUD_CLI} is sgs -q"
```

Comment:
This will list all the sgs in all the vpc, I think we should limit it to the current vpc via `--vpc` parameter, right?

Reply:
in order to check the following selected sgs` 'rhel_worker_sgs="${infra_id}-sg-cluster-wide,${infra_id}-sg-openshift-net"` in the `${IBMCLOUD_CLI} is sgs -q` list

Reply:
pls check my last comments, it is listing all the sg, even belonging to other vpcs, right?
```
[root@preserve-jialiu ~]# ibmcloud is sgs -q
ID                                          Name                                          Rules   Targets   VPC                              Resource group   
r022-1bd41646-b8f7-4d75-9b22-6750a7b85926   ci-op-2386srw3-86507-gklt7-sg-cluster-wide    10      6         ci-op-2386srw3-86507-gklt7-vpc   ci-op-2386srw3-86507-gklt7   
r022-17d03e0e-7f12-4410-98a8-a6ec6f461410   ci-op-2386srw3-86507-gklt7-sg-control-plane   3       3         ci-op-2386srw3-86507-gklt7-vpc   ci-op-2386srw3-86507-gklt7   
r022-5bb12ba7-3b15-461b-b1c5-c10f7ce3e4cc   ci-op-2386srw3-86507-gklt7-sg-cp-internal     2       3         ci-op-2386srw3-86507-gklt7-vpc   ci-op-2386srw3-86507-gklt7   
r022-d781f108-9134-4deb-ac22-560c87597373   ci-op-2386srw3-86507-gklt7-sg-kube-api-lb     4       2         ci-op-2386srw3-86507-gklt7-vpc   ci-op-2386srw3-86507-gklt7   
r022-362a2c65-27e0-4962-b3d5-21131df0edb8   ci-op-2386srw3-86507-gklt7-sg-openshift-net   17      6         ci-op-2386srw3-86507-gklt7-vpc   ci-op-2386srw3-86507-gklt7   
r022-d70ed6bc-7133-4465-9527-76aa99d30931   verify-tattoo-endearing-caption               2       0         ci-op-2386srw3-86507-gklt7-vpc   ci-op-2386srw3-86507-gklt7   
r022-0f2c24f8-e989-4099-bc35-29b5a3c9e547   ci-op-3sw585kb-d810a-bt6nt-sg-cluster-wide    6       6         ci-op-3sw585kb-d810a-bt6nt-vpc   ci-op-3sw585kb-d810a-bt6nt   
r022-30875609-46f8-4d7f-8459-42adc4436b16   ci-op-3sw585kb-d810a-bt6nt-sg-control-plane   3       3         ci-op-3sw585kb-d810a-bt6nt-vpc   ci-op-3sw585kb-d810a-bt6nt   
r022-56d82954-3112-4ba0-97b3-92a6462688ae   ci-op-3sw585kb-d810a-bt6nt-sg-cp-internal     2       3         ci-op-3sw585kb-d810a-bt6nt-vpc   ci-op-3sw585kb-d810a-bt6nt   
r022-2c273b84-4f23-4fc1-825b-fbeaf48debbe   ci-op-3sw585kb-d810a-bt6nt-sg-kube-api-lb     4       2         ci-op-3sw585kb-d810a-bt6nt-vpc   ci-op-3sw585kb-d810a-bt6nt   
r022-f4966b16-8c7d-446d-b574-af8b5b8e989f   ci-op-3sw585kb-d810a-bt6nt-sg-openshift-net   9       6         ci-op-3sw585kb-d810a-bt6nt-vpc   ci-op-3sw585kb-d810a-bt6nt   
r022-69a8d344-7c4c-4827-823b-e698309b4e16   sassy-sticker-purgatory-subsidy               2       0         ci-op-3sw585kb-d810a-bt6nt-vpc   ci-op-3sw585kb-d810a-bt6nt   
r022-04bcc11f-b57f-4fe9-93ac-5fd41a05e152   ci-op-6lix8wcx-1d8dd-nzj6s-sg-cluster-wide    10      6         ci-op-6lix8wcx-1d8dd-nzj6s-vpc   ci-op-6lix8wcx-1d8dd-nzj6s   
r022-2f3d7e53-7aa6-47ea-aa72-1145323a30aa   ci-op-6lix8wcx-1d8dd-nzj6s-sg-control-plane   3       3         ci-op-6lix8wcx-1d8dd-nzj6s-vpc   ci-op-6lix8wcx-1d8dd-nzj6s   
r022-e59069cf-bc20-4b67-b245-88907930de7e   ci-op-6lix8wcx-1d8dd-nzj6s-sg-cp-internal     2       3         ci-op-6lix8wcx-1d8dd-nzj6s-vpc   ci-op-6lix8wcx-1d8dd-nzj6s   
r022-0b5a513c-4634-4fb6-a64e-2002c22246f5   ci-op-6lix8wcx-1d8dd-nzj6s-sg-kube-api-lb     4       2         ci-op-6lix8wcx-1d8dd-nzj6s-vpc   ci-op-6lix8wcx-1d8dd-nzj6s   
r022-a91aad33-8fcf-4289-acb4-4dedd56a21ba   ci-op-6lix8wcx-1d8dd-nzj6s-sg-openshift-net   17      6         ci-op-6lix8wcx-1d8dd-nzj6s-vpc   ci-op-6lix8wcx-1d8dd-nzj6s   
r022-18efe809-69b1-4fe9-9282-91684f59bb6d   confider-bronchial-commotion-valuables        2       0         ci-op-6lix8wcx-1d8dd-nzj6s-vpc   ci-op-6lix8wcx-1d8dd-nzj6s   
r022-aed31dd6-2f6b-43dc-9e48-31144edb917b   ci-op-9jjg7280-ca4da-5n5lm-sg-cluster-wide    10      6         ci-op-9jjg7280-ca4da-5n5lm-vpc   ci-op-9jjg7280-ca4da-5n5lm   
r022-6d833258-432f-4222-86d2-e2c68a3e7e66   ci-op-9jjg7280-ca4da-5n5lm-sg-control-plane   3       3         ci-op-9jjg7280-ca4da-5n5lm-vpc   ci-op-9jjg7280-ca4da-5n5lm   
r022-20b67125-dde3-437c-aaf5-164a67ade478   ci-op-9jjg7280-ca4da-5n5lm-sg-cp-internal     2       3         ci-op-9jjg7280-ca4da-5n5lm-vpc   ci-op-9jjg7280-ca4da-5n5lm   
r022-a1ac6a45-b897-425c-b09d-3e068e64ae53   ci-op-9jjg7280-ca4da-5n5lm-sg-kube-api-lb     4       2         ci-op-9jjg7280-ca4da-5n5lm-vpc   ci-op-9jjg7280-ca4da-5n5lm   
r022-53295cd8-a816-493d-a2b2-e9ea768b80e8   ci-op-9jjg7280-ca4da-5n5lm-sg-openshift-net   17      6         ci-op-9jjg7280-ca4da-5n5lm-vpc   ci-op-9jjg7280-ca4da-5n5lm   
r022-63e18de9-1031-43ef-b33b-6128417a6a15   ideology-chosen-handiwork-shortcake           2       0         ci-op-9jjg7280-ca4da-5n5lm-vpc   ci-op-9jjg7280-ca4da-5n5lm   
r022-3fe6064e-4237-4ae9-9a29-62f230239cc0   ci-op-xi02mx1g-0ae3e-6gzwr-sg-cluster-wide    10      0         ci-op-xi02mx1g-0ae3e-vpc         ci-op-xi02mx1g-0ae3e-rg   
r022-d3486f26-460e-47e3-bc98-c1da23f433d2   ci-op-xi02mx1g-0ae3e-6gzwr-sg-control-plane   3       0         ci-op-xi02mx1g-0ae3e-vpc         ci-op-xi02mx1g-0ae3e-rg   
r022-ceb412d1-a1d7-4626-9951-95a697d954c0   ci-op-xi02mx1g-0ae3e-6gzwr-sg-cp-internal     2       0         ci-op-xi02mx1g-0ae3e-vpc         ci-op-xi02mx1g-0ae3e-rg   
r022-b34241d4-3686-4f6b-9ce3-ff4ac848e95d   ci-op-xi02mx1g-0ae3e-6gzwr-sg-kube-api-lb     4       2         ci-op-xi02mx1g-0ae3e-vpc         ci-op-xi02mx1g-0ae3e-rg   
r022-102a4d3a-5a19-41d3-8cd6-bfa34b0817ec   ci-op-xi02mx1g-0ae3e-6gzwr-sg-openshift-net   17      0         ci-op-xi02mx1g-0ae3e-vpc         ci-op-xi02mx1g-0ae3e-rg   
r022-687b7b64-c842-4bf0-b852-01a7e04a9458   wolverine-issue-bleach-customs                8       1         ci-op-xi02mx1g-0ae3e-vpc         ci-op-xi02mx1g-0ae3e-rg   
r022-f8321471-fbd7-4386-bc2d-3b717dea1c91   brain-levitate-flattery-demanding             8       1         ci-op-zvwgjb8d-324f6-vpc         ci-op-zvwgjb8d-324f6-rg   
r022-74963cbf-2d3b-4cc4-941b-e10e666d0373   ci-op-zvwgjb8d-324f6-8v7q5-sg-cluster-wide    10      6         ci-op-zvwgjb8d-324f6-vpc         ci-op-zvwgjb8d-324f6-8v7q5   
r022-8ee797e1-e4d4-4c6b-a9dd-5ee9b3b94591   ci-op-zvwgjb8d-324f6-8v7q5-sg-control-plane   3       3         ci-op-zvwgjb8d-324f6-vpc         ci-op-zvwgjb8d-324f6-8v7q5   
r022-cd18a3ed-ab5c-4a77-a56a-3f6172db047d   ci-op-zvwgjb8d-324f6-8v7q5-sg-cp-internal     2       3         ci-op-zvwgjb8d-324f6-vpc         ci-op-zvwgjb8d-324f6-8v7q5   
r022-e30d07c7-c1bb-40f9-93f3-63968ab8c0d6   ci-op-zvwgjb8d-324f6-8v7q5-sg-kube-api-lb     4       1         ci-op-zvwgjb8d-324f6-vpc         ci-op-zvwgjb8d-324f6-8v7q5   
r022-ffd066a2-7046-4bac-8f92-449df96bbfbd   ci-op-zvwgjb8d-324f6-8v7q5-sg-openshift-net   17      6         ci-op-zvwgjb8d-324f6-vpc         ci-op-zvwgjb8d-324f6-8v7q5   
r022-d522bb5c-9f8f-4883-9ace-20dfbaac63e4   ethanol-track-wiry-onshore                    4       0         jp-tok-default-vpc               Default
```

Reply:
[workers-rhel-ibmcloud-provision/build-log.txt](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/51226/rehearse-51226-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.16-installer-rehearse-ibmcloud/1783130903896133632/artifacts/installer-rehearse-ibmcloud/workers-rhel-ibmcloud-provision/build-log.txt) after `${IBMCLOUD_CLI} target -g ${cluster_rg}`,  then `run_command "${IBMCLOUD_CLI} is sgs -q"` will just show the sgs in the ${cluster_rg}